pkg/products/amqonline/prometheusRules.go

```diff
@@ -0,0 +1,70 @@
+package amqonline
+
+import (
+	"context"
+	"fmt"
+	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/config"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func (r *Reconciler) reconcileKubeStateMetricsAmqOnline(ctx context.Context,serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error){
+
+	monitoringConfig := config.NewMonitoring(config.ProductConfig{})
+ 	rule := &monitoringv1.PrometheusRule{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:		"ksm-amqonline-alerts",
+			Namespace:  r.Config.GetNamespace(),
+		},
+}
+
+	var namespace = r.Config.GetNamespace()
+
+	rules := []monitoringv1.Rule{
+		{
+			Alert: "AMQOnlinePodCount",
+			Annotations: map[string]string{
+				"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md",
```

Comment:
The SOP url, can be moved to global const, making changes and maintenance would be easy. WDYT?

Reply:
i was holding off to rebase from Robs Pr and convert them but i can do it now